src/crimson/osd/osd_operations/client_request.cc

```diff
@@ -36,7 +36,7 @@ ClientRequest::~ClientRequest()
 
 void ClientRequest::print(std::ostream &lhs) const
 {
-  lhs << *m;
+  lhs << "m=" << *m << "prev_op_id=" << prev_op_id;
```

Comment:
how about adding a space before "prev_op_id"?

Reply:
Sure, will fix that just after agreeing on the debug case.